File: engine/engine-model/src/main/scala/tools/packet/ui/FinishedSort2.scala
package tools.packet.ui

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class FinishedSort2(private var _inventory: Int) extends PacketInput {
  def inventory: Int = _inventory

  override def opcode(): SendOpcode = SendOpcode.SORT_ITEM_RESULT
}

File: engine/engine-model/src/main/java/tools/packet/ui/FinishedSort2.java
package tools.packet.ui;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record FinishedSort2(int inventory) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SORT_ITEM_RESULT;
   }
}